tests/integration/helpers/helpers.py

```diff
@@ -222,7 +222,7 @@ async def scale_application(ops_test: OpsTest, application_name: str, scale: int
         await ops_test.model.wait_for_idle(
             apps=[application_name],
             status="active",
-            timeout=1000,
+            timeout=2000,
```

Comment:
Tests frequently timed out here. `1500` was sufficient for most tests except for the db-admin suite.